README.md

```diff
@@ -78,7 +78,11 @@ default (`rustc`, `rust-std`, `cargo`, and `rust-docs`) plus `rustfmt` and
 `clippy`. This profile will be used by `rustup` by default, and it's the one
 recommended for general use.
 * The **complete** profile includes all the components available through
-`rustup`, including `miri` and IDE integration tools (`rls` and `rust-analysis`).
+`rustup`. This should never be used, as it includes *every* component ever included
+in the metadata and thus will almost always fail. If you are looking for a way
+to install devtools such as `miri` or IDE integration tools (`rls`, `rust-analysis`),
+you should use the `default` profile and install the needed additional components
+manually by running `rustup component add`.
```

Comment:
Another way to add the components is to use the -c option: `rustup toolchain add nightly -c rls -c rust-analysis` for instance. I don't know if its worth mentioning that, but if folk are looking for a one-shot way to make it happen, we have it already.

Reply:
I think it is a bit long for a short description of profiles in a README, but it is definitely worth discussing

Reply:
You could improve the wording to satisfy the `-c` stuff to with something like:
```suggestion
manually, either by using `rustup component add` or by using `-c` when installing
the toolchain.
```